finetuner/__init__.py

```diff
@@ -517,6 +519,29 @@ def build_model(
         )
 
 
+def download_huggingface_model(model_name: str, token: Optional[str] = None) -> str:
+    """Download a model from the HuggingFace Hub.
+
+    :param model_name: Either a URL or a model identifier from the HuggingFace Hub.
+    :param token: Optional token to access private models.
+    :return: The local path to the downloaded model.
+    """
+    from huggingface_hub import hf_hub_download, list_repo_files
```

Comment:
where is this requirement?

Reply:
we have transformers already as a dependency which depends on it. Should we add it anyway?